src/lasdi/param.py

```diff
@@ -78,6 +118,40 @@ def createTestGridSpace(self, param_list):
         mesh_grids = self.createHyperMeshGrid(paramRanges)
         return gridSizes, mesh_grids, self.createHyperGridSpace(mesh_grids)
     
+    def createTestGridSpaceForHull(self, param_list):
+        '''
+        This is similar to createTestGridSpace, but with some different variables.
+        We take the min/max value of each parameter, and create a uniform rectangular grid
+        over the parameter space with 'sample_size' points in each dimension. 
+        '''
+
+        paramRanges = []
+        gridSizes = []
+
+        for param in param_list:
+            Nx, _ = getParam1DSpace[param['test_space_type']](param)
+            minval = param['min']
+            maxval = param['max']
+            gridSizes += [Nx]
+            paramRanges += [np.linspace(minval, maxval, Nx)]
+
+        mesh_grids = self.createHyperMeshGrid(paramRanges)
+        return gridSizes, mesh_grids, self.createHyperGridSpace(mesh_grids)
+    
+    def createTestSpaceFromHull(self, param_list):
```

Comment:
It would be great if you can add comments for the new functions you wrote, following the numpy style.